idaes/models/properties/modular_properties/transport_properties/tests/test_shell_and_tube_1D_transport.py

```diff
@@ -0,0 +1,516 @@
+            )
+            iscale.set_scaling_factor(
+                m.fs.unit.cold_side_heat_transfer_coefficient[t, x], 1e-2
+            )
+            iscale.constraint_scaling_transform(
+                m.fs.unit.tube_heat_transfer_coeff_eqn[t, x], 1e-2
+            )
+        for x in m.fs.unit.hot_side.length_domain:
+            iscale.set_scaling_factor(m.fs.unit.hot_side.heat[t, x], 1e-3)
+            iscale.set_scaling_factor(m.fs.unit.shell_reynolds_number[t, x], 1e-3)
+            iscale.constraint_scaling_transform(
+                m.fs.unit.shell_reynolds_number_eqn[t, x], 1e-3
+            )
+            iscale.set_scaling_factor(
+                m.fs.unit.hot_side_heat_transfer_coefficient[t, x], 1e-2
+            )
+            iscale.constraint_scaling_transform(
+                m.fs.unit.shell_heat_transfer_coeff_eqn[t, x], 1e-2
+            )
+
+    iscale.calculate_scaling_factors(m)
+
+    return m
+
+
+def initialize_model(m):
+    m.fs.unit.cold_side_heat_transfer_coefficient.fix(51000)
+    m.fs.unit.tube_reynolds_number.fix()
+    m.fs.unit.tube_reynolds_number_eqn.deactivate()
+    m.fs.unit.tube_heat_transfer_coeff_eqn.deactivate()
+    m.fs.unit.hot_side_heat_transfer_coefficient.fix(2000)
+    m.fs.unit.shell_reynolds_number.fix()
+    m.fs.unit.shell_reynolds_number_eqn.deactivate()
+    m.fs.unit.shell_heat_transfer_coeff_eqn.deactivate()
+
+    m.fs.unit.initialize_build()
+
+    m.fs.unit.cold_side_heat_transfer_coefficient.unfix()
+    m.fs.unit.tube_reynolds_number.unfix()
+    m.fs.unit.tube_reynolds_number_eqn.activate()
+    m.fs.unit.tube_heat_transfer_coeff_eqn.activate()
+    m.fs.unit.hot_side_heat_transfer_coefficient.unfix()
+    m.fs.unit.shell_reynolds_number.unfix()
+    m.fs.unit.shell_reynolds_number_eqn.activate()
+    m.fs.unit.shell_heat_transfer_coeff_eqn.activate()
+    for t in m.fs.time:
+        for x in m.fs.unit.cold_side.length_domain:
+            calculate_variable_from_constraint(
+                m.fs.unit.tube_reynolds_number[t, x],
+                m.fs.unit.tube_reynolds_number_eqn[t, x],
+            )
+            calculate_variable_from_constraint(
+                m.fs.unit.cold_side_heat_transfer_coefficient[t, x],
+                m.fs.unit.tube_heat_transfer_coeff_eqn[t, x],
+            )
+        for x in m.fs.unit.hot_side.length_domain:
+            calculate_variable_from_constraint(
+                m.fs.unit.shell_reynolds_number[t, x],
+                m.fs.unit.shell_reynolds_number_eqn[t, x],
+            )
+            calculate_variable_from_constraint(
+                m.fs.unit.hot_side_heat_transfer_coefficient[t, x],
+                m.fs.unit.shell_heat_transfer_coeff_eqn[t, x],
+            )
+    return solver.solve(m, tee=True)
+
+
+class Test_transport_properties_ideal(object):
+    @pytest.fixture(scope="class")
+    def hx(self):
+        return build_model(eos=EosType.IDEAL)
+
+    @pytest.mark.component
```

Comment:
These are really `intergration` tests, as they involve combining a unit model and a property package. This will also help keep test times more manageable. This also ties into a TODO comment I noted later about changing the marks when the new NL writer was the default.